python/tvm/script/ir_builder/tir/frame.py

```diff
@@ -38,6 +38,11 @@ class BlockFrame(TIRFrame):
     ...
```

Comment:
please add mypy checks for our IRBuilder package:

In file `tests/scripts/task_mypy.sh`, let's add:

```python
echo "Checking MyPy Type defs in the tvmscript IRBuilder package."
mypy  --check-untyped-defs python/tvm/script/ir_builder
```

Reply:
added